index.html

```diff
@@ -289,6 +289,20 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact-form">
+        <div class="description-text">
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </div>
+        <div class="form">
+          <form>
+            <input type="text" required maxlength="20" placeholder="Full name">
```

Comment:
- [ ] Greate Job here. The only issue here is that the name field should accepts up to 30 chars as mentioned in the requirements